sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -94,10 +94,7 @@ export interface EventDataAdapterParameters {
 // @public
 export interface EventDataBatch {
     readonly count: number;
-    // @internal
-    _generateMessage(): Buffer;
     readonly maxSizeInBytes: number;
-    // @internal
     readonly _messageSpanContexts: SpanContext[];
```

Comment:
Isn't `SpanContext` still in beta?

Reply:
It's imported from core-tracing.  This code has been here for a few months already.  Anyway, Event Hubs package is currently in beta too.

Reply:
Ah, I see. I still wonder if it would be possible to remove this from public surface, but I'm not sure I understand the scenario

Reply:
Looks that it is for internal use, I will try hiding it.